src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/Resources/Strings.resx

```diff
@@ -231,4 +231,10 @@
   <data name="MalformedFormatStringsTitle" xml:space="preserve">
     <value>Logging method contains malformed format strings</value>
   </data>
+  <data name="LoggingUnsupportedLanguageVersionTitle" xml:space="preserve">
+    <value>C# language version not supported by the source generator.</value>
+  </data>
+  <data name="LoggingUnsupportedLanguageVersionMessageFormat" xml:space="preserve">
+    <value>The Logging source generator is not available in C# '{0}'. Please use language version '{1}' or greater.</value>
```

Comment:
I think we should be removing quotes from both places rather than adding them. We control the version enum that is being passed in both cases, so there's no need to guard with quotes.

I'll submit a new PR changing this.

Reply:
I am not seeing having the quotes is bad to show in the message.

Reply:
Quotes are typically used to delimit inputs that might contain arbitrary values, including whitespace. We control the inputs passed to the diagnostic and it's kind of weird to say `C# '8'`.

Reply:
I created a PR but somehow didn't add you as a reviewer:

https://github.com/dotnet/runtime/pull/90720

Reply:
> Quotes are typically used to delimit inputs that might contain arbitrary values, including whitespace. We control the inputs passed to the diagnostic and it's kind of weird to say C# '8'.

Not necessary, sometimes the quotes can be used to emphasize the info inside the quote. Anyway, I don't mind either way. I'll take a look at the PR.